release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3843,3 +3843,37 @@ $ oc adm release info 4.11.53 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-11-54"]
+=== RHSA-2023:7479 {product-title} 4.11.54 bug fix update and security update
+
+Issued: 2023-11-29
+
+{product-title} release 4.11.54, which includes security updates, is now available. Bug fixes included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2023:7479[RHSA-2023:7479] advisory. RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:7481[RHSA-2023:7481] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.11.54 --pullspecs
+----
+
+[id="ocp-4-11-54-features"]
+==== Features
+
+[id="ocp-4-11-54-APIserver-tracked-by-Insights-Operator"]
+===== APIServer.config.openshift.io is now tracked by Insights Operator
+
+After running the Insights Operator, a new file is now available in the archive in the path `config/apiserver.json` with the information about the audit profile.
++
+Access to audit profiles allows you to understand what audit policy is common practice, what profiles are most commonly used, what differences there are between industries, and what kind of customisation is applied.
```

Comment:
"Allows" is a form of anthropomorphism (snippet from IBM SG):

![Screenshot from 2023-11-29 10-41-35](https://github.com/openshift/openshift-docs/assets/57954076/3da240cc-67a3-444f-8f1d-56b8eaacec1f)

How about:

> By accessing audit profiles, you can determine what audit policy meets common practice, commonly adopted profiles, differences that exist between profiles adopted by industries, and levels of customisation for profiles.

Reply:
Good catch!

Reply:
I found the same feature in a 4.13 z release so I'm altering this to match it.